storage/StorageInterface.h

```diff
@@ -214,6 +214,23 @@ namespace storage
 
 
     /**
+     * Abstract class for some callbacks during commit. Currently only works
+     * for btrfs as root filesystem.
+     */
+    class CommitCallbacks
+    {
+    public:
+
+	virtual ~CommitCallbacks() {}
+
+	virtual void post_root_filesystem_create() const {}
+	virtual void post_root_mount() const {}
+	virtual void post_root_fstab_add() const {}
```

Comment:
It just simplify code as getter return always valid object, so it is slightly cleaner and less error prone as you do not need to care if it is null pointer leading to segfault.

As I stated in comment, it is just idea to improvement and do not block LGTM.